File: app/src/main/java/com/permissionnanny/missioncontrol/PermissionInfoViewHolder.java
package com.permissionnanny.missioncontrol;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.Spinner;
import android.widget.TextView;
import butterknife.Bind;
import butterknife.ButterKnife;
import com.permissionnanny.R;

public class PermissionInfoViewHolder extends RecyclerView.ViewHolder {
    @Bind(R.id.permissionName) TextView tvPermissionName;
    @Bind(R.id.permissionDesc) TextView tvPermissionDesc;
    @Bind(R.id.permissionAccess) Spinner sPermissionAccess;

    public PermissionInfoViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/kotlin/com/permissionnanny/missioncontrol/PermissionInfoViewHolder.kt
package com.permissionnanny.missioncontrol

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.Spinner
import android.widget.TextView
import com.permissionnanny.R
import org.jetbrains.anko.find

class PermissionInfoViewHolder(itemView: View)
    : RecyclerView.ViewHolder(itemView) {

    internal val tvPermissionName: TextView = itemView.find(R.id.permissionName)
    internal var tvPermissionDesc: TextView = itemView.find(R.id.permissionDesc)
    internal var sPermissionAccess: Spinner = itemView.find(R.id.permissionAccess)
}
